### PR TITLE
Fixed to be able to set "personium.example.com" with Vagrantfile

### LINKED ISSUE #35: Use proper FQDN instead of localhost
Some Personium tools/apps block localhost. Therefore, it is better to use a dummy FQDN and provide method for host PC configuration.

### PR DESCRIPTION
**Pull request by Keita-Nakagama**
close #35

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -66,11 +66,11 @@ Vagrant.configure(2) do |config|
...
-  # if Vagrant.has_plugin?("vagrant-proxyconf")
-  #   config.proxy.http = "http://username:password@host:port/"
-  #   config.proxy.https = "http://username:password@host:port/"
-  #   config.proxy.no_proxy = "localhost,127.0.0.1"
-  # end
+  if Vagrant.has_plugin?("vagrant-proxyconf")
+    config.proxy.http = "http://fj1738bt:0019970807@kmt.proxy.nic.fujitsu.com:8080"
+    config.proxy.https = "http://fj1738bt:0019970807@kmt.proxy.nic.fujitsu.com:8080"
+    config.proxy.no_proxy = "personium.example.com,127.0.0.1"
+  end
```
Comment (KourokiTakuto) [inline Vagrantfile:74]: Please fix only "localhost".
Please check the difference before the pull request.

**Review by KourokiTakuto — CHANGES_REQUESTED** [review]

**Review by KourokiTakuto — COMMENTED** [review]
Please correct personium-unit-config.properties.j2.

**Review by dixonsiu — APPROVED** [review]

File: Vagrantfile (hunk 1 above)
Comment (Keita-Nakagama) [reply Vagrantfile:74]: Fixed it.

**Review by KourokiTakuto — APPROVED** [review]

**Review by hiroaki-shibata — APPROVED** [review]
I confirmed and approve.

**Pull request merged**